terraform/modules/sync_check/service/sync_check.rb

```diff
@@ -51,8 +51,8 @@ def get_and_assert_env_variable(name)
   client.post_message "✅ Sync check for #{hostname} passed. 🌲🌳🌲🌳🌲"
 else
   client.post_message "⛔ Sync check for #{hostname} fiascoed. 🔥🌲🔥"
-  SyncCheck.new.run_forest_cli("--chain #{network} db clean --force")
-  logger.info 'DB Cleaned'
+  SyncCheck.new.run_forest_cli("db destroy --chain #{network} --force")
```

Comment:
It would be best to have it as a separate PR. Sync check runs on the `edge` tag, which will get pushed as soon as the re-org PR is merged in Forest.

Reply:
I will keep this PR with the updated Sync check. Should I remove my modification to the snapshot upload as well?